resources.yaml

```diff
@@ -80,7 +80,7 @@ deployment:
     image: htcondor-secondary
     secondary_htcondor_cluster: true
   worker-c28m225-htcondor-secondary:
-    count: 0 #7
+    count: 7
```

Comment:
This flavor is used only for the training. We do not use them for our regular queue. So, this should be `0`.